lib/util/git/index.ts

```diff
@@ -900,7 +904,27 @@ export async function mergeBranch(branchName: string): Promise<void> {
       ]),
     );
     status = await git.status();
-    await gitRetry(() => git.merge(['--ff-only', branchName]));
+    if (
+      mergeStrategy === 'fast-forward' ||
+      mergeStrategy === 'rebase' ||
+      mergeStrategy === 'auto'
+    ) {
+      await gitRetry(() => git.merge(['--ff-only', branchName]));
+    } else if (mergeStrategy === 'merge-commit') {
+      let commandOptions = ['--no-ff', '--no-edit'];
+      if (commitMessage) {
+        commandOptions = commandOptions.concat(['-m', commitMessage]);
+      }
```

Comment:
If it's not supplied then does git choose its own default message?

Reply:
Yes exactly. For me it creates a commit message in the format `Merge branch '<branch>'`. According to the documentation from git, this comes from `git fmt-merge-msg`: https://git-scm.com/docs/git-fmt-merge-msg